### DISCUSSION TITLE
Warning: Failed prop type: Invalid prop `checked` supplied to `Checkbox`.

### DISCUSSION THREAD

**Discussion by llhxhal** [ant-design/ant-design#30491] [Q&A]
checkbox选中后警告

**Comment by yoyo837** [comment]
上复现

**Reply by yoyo837** [reply]
粘一堆没用，也基本不看，也无法运行，在 https://codesandbox.io/s/nvyzg 这里修改吧，改好后保存，新的链接发出来

**Reply by llhxhal** [reply]
不好意思久等了，处理了一些bug，现在能跑起来
https://t5bxy.csb.app/

**Reply by yoyo837** [reply]
不要发预览链接，发代码链接，不然别人没法改。

**Reply by llhxhal** [reply]
https://codesandbox.io/s/epic-firefly-t5bxy?file=/index.js

**Reply by yoyo837** [reply]
卡彪了，先解决下性能问题吧。
